README.md

```diff
@@ -10,7 +10,7 @@ npm install stringify
 
 ## Usage ##
 
-Setup Browserify to use this middleware in your app:
+Use it directly from the command line `browserity -t stringify myfile.js` or setup Browserify to use this middleware in your app:
```

Comment:
Ok. Should I do that?

/Kenneth

Den 20/01/2015 kl. 08.40 skrev John Postlethwait <notifications@github.com>:

In README.md
<https://github.com/JohnPostlethwait/stringify/pull/13#discussion-diff-23206539>
:

> @@ -10,7 +10,7 @@ npm install stringify
>
>  ## Usage ##
>
> -Setup Browserify to use this middleware in your app:
> +Use it directly from the command line `browserity -t stringify myfile.js` or setup Browserify to use this middleware in your app:

Nit: It'd be better to make this a new section I think.

### Command Line ###
`browserity -t stringify myfile.js`
### Browserify Middleware ###

Setup Browserify to use this middleware in your app:
...

—
Reply to this email directly or view it on GitHub
<https://github.com/JohnPostlethwait/stringify/pull/13/files#r23206539>.